Hazelcast.Net/Hazelcast.Config/ClientConfig.cs

```diff
@@ -79,30 +82,55 @@ public class ClientConfig
 
         private SerializationConfig _serializationConfig = new SerializationConfig();
 
+        /// <summary>
+        /// Helper method to add a new ListenerConfig
```

Comment:
I guess it is best to user periods at the end of sentences that are in-between the "summary" tags.